src/video.ts

```diff
@@ -21,19 +23,27 @@ const parseVideo = (element: BlockElement, atoms?: Atoms): Option<Video> => {
 		return none;
 	}
 
-	const { posterUrl, duration, assets, activeVersion } = atom.data.media;
+	const {
+		posterUrl,
+		duration,
+		assets,
+		activeVersion,
+		title,
+	} = atom.data.media;
 	const videoId = assets.find(
 		(asset) => asset.version.toNumber() === activeVersion?.toNumber(),
 	)?.id;
 
-	if (!posterUrl || !videoId) {
+	if (!posterUrl || !videoId || !id) {
```

Comment:
Slightly pedantic (sorry!) but if we call this `atomId` rather than just `id` throughout it makes it a bit easier to follow. Had to go back through the file to see where it was coming from.